src/log.c

```diff
@@ -192,7 +189,7 @@ void
 log_msg(log_level_t level, const char* const area, const char* const msg)
 {
     if (level >= level_filter && logp) {
-        dt = g_date_time_new_now(tz);
+        dt = g_date_time_new_now_local();
```

Comment:
does `dt` really have to have global storage? should be sufficient to do 
```suggestion
        GDateTime *dt = g_date_time_new_now_local();
```
and after usage `g_date_time_unref(dt)`